File: game.py
import random

def evaluate_guess(secret_number, attempts):
    user_guess = input("Enter a number: ")
    try:
        guess = int(user_guess)
    except ValueError as error:
        print("Only integer numbers are accepted.")
        print("Try again!")
        quit()
    if guess < secret_number:
        print("The secret number is bigger!")
        success = False
    elif guess > secret_number:
        print("The secret number is smaller!")
        success = False
    elif guess == secret_number:
        print("Congratulation! You found the secret number in " + str(attempts) + " attempts!")
        success = True
    return success



print("Welcome in the guess the number game!")
print("You have to find the secret number which is between 1 and 100.")
secret_number = random.randint(1, 100)
attempts = 0

while True:
    attempts += 1
    success = evaluate_guess(secret_number, attempts)
    if success:
        break

print("Try again!")

File: game.js
function evaluateGuess(secretNumber, attempts) {
    var userGuess = window.prompt("Enter a number:");
    var guess = Number(userGuess);
    if (Number.isInteger(guess)) {
        if (guess < secretNumber) {
            console.log("The secret number is bigger!");
            return false;
        } else if (guess > secretNumber) {
            console.log("The secret number is smaller!");
            return false;
        } else if (guess == secretNumber) {
            if (attempts == 1) {
                var attemptWord = " attempt!";
            } else {
                var attemptWord = " attempts!";
            }
            console.log("Congratulation! You found the secret number in " + attempts.toString() + attemptWord);
            return true;
        }
    } else {
        console.log("Only integer numbers are accepted.");
        return true;
    }
}


console.log("Welcome in the guess the number game!");
console.log("You have to find the secret number which is between 1 and 100.");
var secretNumber = Math.floor(Math.random() * 100 + 1);
var attempts = 0;

while (true) {
    attempts++;
    if (evaluateGuess(secretNumber, attempts)) {
        break;
    }
}

console.log("Try again!");
